spec/validations/manifest_spec.rb

```diff
@@ -4,6 +4,7 @@
 require 'tmpdir'
 
 describe ZendeskAppsSupport::Validations::Manifest do
+
```

Comment:
nit: it seems other specs don't have this additional line break. How about reverting this one and removing the the additional line break on the new test examples?